app/models/attachment.js

```diff
@@ -219,9 +219,29 @@ export function addModel(dbAdapter) {
       this.fileExtension = supportedImageTypes[this.mimeType]
       this.noThumbnail = '1' // this may be overriden below
 
-      // Store original image size
       let originalImage = promisifyAll(gm(tmpAttachmentFile))
```

Comment:
Oh yeah, since we are using that promisified variable for rotation, it seems impossible to optimize it this way. Disregard my suggestion then.